tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/internal/EditorMicroservice.java

```diff
@@ -1214,6 +1214,18 @@ public Response exportApps(
                             .entity(errorMessage)
                             .build();
                 }
+                if (!dockerBuildConfig.getImageName().equals(
+                                dockerBuildConfig.getImageName().toLowerCase())
+                ) {
```

Comment:
Format

Reply:
Fixed in https://github.com/siddhi-io/distribution/pull/752/commits/9956b3889430f9b88bf867471d04b547ae4f29b9